### PR TITLE
Adding mj-style

### PR DESCRIPTION
**Pull request by arthurkatz**
Following MJML 3.0.0.beta release

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "language-mjml",
-  "version": "1.6.0",
+  "version": "1.7.0",
```
Comment (ngarnier) [inline package.json]: Please do not update the version in package.json, Atom does it automatically on publish

**Review by ngarnier — CHANGES_REQUESTED** [review]
Thanks @arthurkatz, providing updates even before the MJML version is released... you're amazing 🙌

**Comment by arthurkatz** [comment]
Oh I didn't knew, I'm fixing this ;)

Le jeudi 6 octobre 2016, Nicolas Garnier notifications@github.com a
écrit :

**Comment by arthurkatz** [comment]
@ngarnier fixed

**Comment by arthurkatz** [comment]
Next step is to add CSS colors into mj-style's content

**Comment by ngarnier** [comment]
hey @arthurkatz, thanks again. I just tested it as we're about to release MJML v3 but it seems there's an issue. When typing `style`+tab, the component unwrapped is <mj-chart />, as seen here: 



Anyway, we haven't merged `mj-chart` yet so it'd be better if you can just remove it for now.

**Comment by arthurkatz** [comment]
Fixed !

**Comment by ngarnier** [comment]
Looking good, merging 🚀 ! Thanks Arthur

The plugin is up to date & available on Atom!

**Pull request merged**